rules/windows/credential_access_wbadmin_ntds.toml

```
@@ -0,0 +1,83 @@
+[metadata]
+creation_date = "2024/06/05"
+integration = ["windows", "endpoint", "system"]
+maturity = "production"
+updated_date = "2024/06/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of wbadmin to access the NTDS.dit file in a domain controller. Attackers with privileges from
+groups like Backup Operators can abuse the utility to perform credential access and compromise the domain.
+"""
+from = "now-9m"
+index = [
+    "winlogbeat-*",
+    "logs-endpoint.events.process-*",
+    "logs-windows.*",
+    "endgame-*",
+    "logs-system.security*",
+]
+language = "eql"
+license = "Elastic License v2"
+name = "NTDS Dump via Wbadmin"
+references = [
+    "https://medium.com/r3d-buck3t/windows-privesc-with-sebackupprivilege-65d2cd1eb960"
+]
+risk_score = 47
+rule_id = "d93e61db-82d6-4095-99aa-714988118064"
+severity = "medium"
+tags = [
+    "Domain: Endpoint",
+    "OS: Windows",
+    "Use Case: Threat Detection",
+    "Tactic: Credential Access",
+    "Data Source: Elastic Endgame",
+    "Data Source: Elastic Defend"
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where host.os.type == "windows" and event.type == "start" and
+    process.name : "wbadmin.exe" and process.args : "recovery" and process.args : "*ntds.dit*"
```

for performance maybe just we limit to process creation (the proc will be logged first anyway):
```suggestion
process where host.os.type == "windows" and event.type == "start" and
    (process.name : "wbadmin.exe" or ?process.pe.original_file_name : "wbadmin.exe") and 
     process.args : "recovery" and process.command_line : "*ntds.dit*"
```